timeseries/tests/smoketests/test_all_models.py

```diff
@@ -182,8 +181,43 @@ def test_all_models_can_handle_all_covariates(
     assert predictions.index.equals(future_test_data.index)
 
 
-@pytest.mark.parametrize("freq", DEFAULT_SEASONALITIES.keys())
-def test_all_models_handle_all_pandas_frequencies(freq, all_model_hyperparams):
+@pytest.mark.parametrize(
+    "freq",
+    [
+        "YE",
+        "QE",
+        "SME",
+        "W",
+        "D",
+        "B",
+        "bh",
+        "h",
+        "min",
+        "s",
+    ],
```

Comment:
Nit: If we are specifying these manually, I would suggest trying some nonstandard combinations like `2W`, `3D` `4h`, `100s`

Reply:
replacing some with these (not adding new) as this leads to a proportionate increase in runtime